
float64 stop_dist #Stopping Distance 

---

bool Status #Execution Status
